src/system_simulation/simulation_runner.rb

```diff
@@ -1,111 +1,160 @@
-#!/usr/bin/env ruby
-
 require 'csv'
 require_relative './animation_data_sample.rb'
 require 'open3'
+require 'singleton'
+require 'benchmark'
 
 require 'fileutils'
 require 'tmpdir'
 
+# This class encapsulates the way of how system simulations are run. Right now we use Modelica and compile / simulate
```

Comment:
Maybe add what you mean by system simulation...
If you don't know anything about the spring project that's hard to get.

Reply:
Done in 6ee258a3c58b1fd820677f52efe4907d283251b1